fabric-website can not start

<!-- 🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨

We’re here to help, but we need a few things from you! 

Please help us reduce duplicates by doing the following steps before logging an issue:
  * Search: https://github.com/OfficeDev/office-ui-fabric-react/search?type=Issues
  * Search by area or component: https://github.com/OfficeDev/office-ui-fabric-react/issues/labels
  
Please provide a reproduction of the bug in a codepen, if possible. Here’s how:
  * Goto https://aka.ms/fabricpen for a starting codepen 
  * You can also use the "Export to Codepen" feature for the various components in our documentation site.
  * Alternatively, you can also use https://aka.ms/fabricdemo to get permanent repro links if the repro occurs with an example. 
    (A permanent link is preferable to "use the website" as the website can change)
 
Note that if you do not provide enough information to reproduce the issue, we may not be able to take action on your report.
-->

### Environment Information 
 
 <!-- 
  ** If using the website, note that the website is often behind. You can find the current version the website uses at the home page (i.e. at the landing page of uifabric.io).
 -->
- __Package version(s)__: git master
- __Browser and OS versions__:  Windows 10

### Please provide a reproduction of the bug in a codepen:
 
```
git clone https://github.com/OfficeDev/office-ui-fabric-react.git
npm install
cd apps/fabric-website/
npm start
```
 
#### Actual behavior:
 
```
ERROR in ./src/components/Nav/Nav.tsx
Module not found: Error: Can't resolve '@uifabric/experiments' in 'C:\Users\liuyang\ReactProjects\office-ui-fabric-react\apps\fabric-website\src\components\Nav'
 @ ./src/components/Nav/Nav.tsx 8:20-52
 @ ./src/components/App/App.tsx
 @ ./src/root.tsx
 @ multi C:/Users/liuyang/ReactProjects/office-ui-fabric-react/common/temp/node_modules/.registry.npmjs.org/webpack-dev-server/3.1.4/webpack@4.7.0/node_modules/webpack-dev-server/client?http://localhost:4321 ./src/root.tsx
i ｢wdm｣: Failed to compile.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/components/CodeBlock/CodeBlock.tsx(3,31):
TS2307: Cannot find module './CodeBlock.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/components/ColorTable/ColorTable.tsx(3,31):
TS2307: Cannot find module './ColorTable.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/components/IconGrid/IconGrid.tsx(3,31):
TS2307: Cannot find module './IconGrid.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/components/Nav/Nav.tsx(11,8):
TS2307: Cannot find module '@uifabric/experiments'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/components/Nav/Nav.tsx(14,31):
TS2307: Cannot find module './Nav.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/components/PageHeader/PageHeader.tsx(3,31):
TS2307: Cannot find module './PageHeader.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/components/Table/Table.tsx(3,31):
TS2307: Cannot find module './Table.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/components/TopNav/TopNav.tsx(8,31):
TS2307: Cannot find module './TopNav.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/BlogPage/BlogPage.tsx(4,31):
TS2307: Cannot find module './BlogPage.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/GetStarted/GetStartedPage.tsx(8,31):
TS2307: Cannot find module './GetStartedPage.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/HomePage/HomePage.tsx(3,31):
TS2307: Cannot find module './HomePage.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/Interstitials/AngularJSPage.tsx(3,31):
TS2307: Cannot find module './Interstitials.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/Interstitials/FabricIOSPage.tsx(2,31):
TS2307: Cannot find module './Interstitials.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/Interstitials/FabricJSPage.tsx(3,31):
TS2307: Cannot find module './Interstitials.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/Overviews/ComponentsPage.tsx(4,31):
TS2307: Cannot find module './Overviews.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/Overviews/StylesPage.tsx(4,31):
TS2307: Cannot find module './Overviews.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/ResourcesPage/ResourcesPage.tsx(4,31):
TS2307: Cannot find module './ResourcesPage.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/Styles/BrandIconsPage/BrandIconsPage.tsx(7,31):
TS2307: Cannot find module './BrandIconsPage.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/Styles/ColorsPage/ColorsPage.tsx(6,31):
TS2307: Cannot find module './ColorsPage.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/Styles/IconsPage/IconsPage.tsx(6,31):
TS2307: Cannot find module './IconsPage.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/Styles/LayoutPage/LayoutPage.tsx(5,31):
TS2307: Cannot find module './LayoutPage.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/Styles/LocalizationPage/LocalizationPage.tsx(6,31):
TS2307: Cannot find module './LocalizationPage.module.scss'.
ERROR in C:/Users/liuyang/ReactProjects/office-ui-fabric-react/apps/fabric-website/src/pages/Styles/TypographyPage/TypographyPage.tsx(5,31):
TS2307: Cannot find module './TypographyPage.module.scss'.
Version: typescript 2.8.4
Time: 14371ms
i ｢wdm｣: Compiling...
  0% compilingi ｢wdm｣: wait until bundle finished: /fabric-sitev5.js

<--- Last few GCs --->

[7020:000002ED84BCEA60]   135577 ms: Mark-sweep 1360.8 (1456.1) -> 1360.7 (1430.1) MB, 622.7 / 0.5 ms  (average mu = 0.109, current mu = 0.001) last resort GC in old space requested
[7020:000002ED84BCEA60]   136222 ms: Mark-sweep 1360.7 (1430.1) -> 1360.6 (1428.1) MB, 644.7 / 0.4 ms  (average mu = 0.056, current mu = 0.000) last resort GC in old space requested


<--- JS stacktrace --->

==== JS stack trace =========================================

    0: ExitFrame [pc: 000002ECCBADC5C1]
    1: StubFrame [pc: 000002ECCBADD9BF]
Security context: 0x0106b211e6e1 <JSObject>
    2: resolve [000001112C262D09] [path.js:~138] [pc=000002ECCC2571B9](this=0x00f452c022a1 <Object map = 00000125AD736449>)
    3: arguments adaptor frame: 1->0
    4: stat [00000186501048B1] [fs.js:~811] [pc=000002ECCC2482AB](this=0x031a03d0e8d1 <Object map = 0000018ED54CA2F1>,path=0x012999639ae9 <String[232]: C:\Us...

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 00007FF7AC69ECF5
 2: 00007FF7AC6781A6
 3: 00007FF7AC678BB0
 4: 00007FF7AC909AAE
 5: 00007FF7AC9099DF
 6: 00007FF7ACE47724
 7: 00007FF7ACE3DE87
 8: 00007FF7ACE3C3FC
 9: 00007FF7ACE45377
10: 00007FF7ACE453F6
11: 00007FF7AC9E84B7
12: 00007FF7ACA8019A
13: 000002ECCBADC5C1
child_process.js:651
    throw err;
    ^

Error: Command failed: node C:\Users\liuyang\ReactProjects\office-ui-fabric-react\scripts\node_modules\webpack-dev-server\bin\webpack-dev-server.js --config C:\Users\liuyang\ReactProjects\office-ui-fabric-react\apps\fabric-website\webpack.serve.config.js --open
    at checkExecSyncError (child_process.js:611:11)
    at execSync (child_process.js:648:13)
    at myExecSync (C:\Users\liuyang\ReactProjects\office-ui-fabric-react\scripts\exec-sync.js:17:18)
    at Object.<anonymous> (C:\Users\liuyang\ReactProjects\office-ui-fabric-react\scripts\start.js:28:3)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
    at Module.load (internal/modules/cjs/loader.js:598:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:741:12)
error An unexpected error occurred: "Command failed.
Exit code: 1
Command: C:\\WINDOWS\\system32\\cmd.exe
Arguments: /d /s /c node ../../scripts/start.js
Directory: C:\\Users\\liuyang\\ReactProjects\\office-ui-fabric-react\\apps\\fabric-website
Output:
".
info If you think this is a bug, please open a bug report with the information provided in "C:\\Users\\liuyang\\ReactProjects\\office-ui-fabric-react\\apps\\fabric-website\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
 
#### Expected behavior:
 
npm run works

--- a/apps/fabric-website/README.md
+++ b/apps/fabric-website/README.md
@@ -10,8 +10,9 @@ Before you get started, make sure you have [node.js](https://nodejs.org/), [gulp
 
 1.  `git clone https://github.com/OfficeDev/office-ui-fabric-react.git` - this is the [Office UI Fabric React project](https://github.com/OfficeDev/office-ui-fabric-react) which you will want to name _office-ui-fabric-react_
 2.  `npm install` the Fabric React project you just cloned in the previous step.
-3.  In the command prompt change directories to the fabric-website director in apps `cd apps/fabric-website/`.
-4.  Then to start the website run `npm start`. This will open your operating system's default web browser with the website. You can make changes to the code which will automatically build and refresh the page using live-reload.
+3.  `npm run buildfast` to build the dependencies of fabric-website
+4.  In the command prompt change directories to the fabric-website director in apps `cd apps/fabric-website/`.
+5.  Then to start the website run `npm start`. This will open your operating system's default web browser with the website. You can make changes to the code which will automatically build and refresh the page using live-reload.
 
 **Note**: Were any steps missing? Could the instructions be made clearer for the next person? Update the instructions above and submit a pull request!
 

--- a/apps/fabric-website/README.md
+++ b/apps/fabric-website/README.md
@@ -8,11 +8,12 @@ Office UI Fabric is a collection of projects that represent the Office and Offic
 
 Before you get started, make sure you have [node.js](https://nodejs.org/), [gulp](http://gulpjs.com/), and [git](https://git-scm.com/) installed. To view the documentation including examples, contracts, current statuses, and to add functionality or fix issues locally, you can:
 
-1.  `git clone https://github.com/OfficeDev/office-ui-fabric-react.git` - this is the [Office UI Fabric React project](https://github.com/OfficeDev/office-ui-fabric-react) which you will want to name _office-ui-fabric-react_
-2.  `npm install` the Fabric React project you just cloned in the previous step.
-3.  `npm run buildfast` to build the dependencies of fabric-website
-4.  In the command prompt change directories to the fabric-website director in apps `cd apps/fabric-website/`.
-5.  Then to start the website run `npm start`. This will open your operating system's default web browser with the website. You can make changes to the code which will automatically build and refresh the page using live-reload.
+1.  `git clone https://github.com/OfficeDev/office-ui-fabric-react.git` - this is the [Office UI Fabric React project](https://github.com/OfficeDev/office-ui-fabric-react)
+2.  Change directories into the newly cloned project's directory: `cd office-ui-fabric-react`
+3.  `npm install` to install dependencies and start building the project
+4.  `npm run buildfast` to build additional dependencies of `fabric-website`
+5.  Change directories to the `fabric-website` directory under `apps`: `cd apps && cd fabric-website`.
+6.  To start the website, run `npm start`. This will open your operating system's default web browser with the website. You can make changes to the code which will automatically build and refresh the page using live-reload.
 
 **Note**: Were any steps missing? Could the instructions be made clearer for the next person? Update the instructions above and submit a pull request!
 

--- a/common/changes/@uifabric/fabric-website/fabric-website_2019-01-14-22-26.json
+++ b/common/changes/@uifabric/fabric-website/fabric-website_2019-01-14-22-26.json
@@ -1,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "packageName": "@uifabric/fabric-website",
+      "type": "none"
+    }
+  ],
+  "packageName": "@uifabric/fabric-website",
+  "email": "elcraig@microsoft.com"
+}

